packages/pages/src/vite-plugin/build/buildStart/rendering/templateUtils.test.tsx

```diff
@@ -16,12 +16,24 @@ const baseTemplateModule: TemplateModuleInternal<any, any> = {
 
 const baseProps: TemplateProps = {
   document: {},
-  __meta: { mode: "development" },
+  __meta: {
+    mode: "development",
+    manifest: {
```

Comment:
Did this need to change?

Reply:
yeah, need to change this to get the test ran properly